template.tpl

```diff
@@ -365,17 +402,28 @@ if (!data.enableFirstPartyCookies) {
 }
 
 
-var eventMetadata = {
-  currency: data.currency,
-  value: data.transactionValue,
-};
+var eventMetadata = {};
+
+if (data.currency) {
+  eventMetadata.currency = data.currency;
+}
+
+if (data.transactionValue) {
+  eventMetadata.value = data.transactionValue;
+}
```

Comment:
```suggestion
var eventMetadata = {
  currency: data.currency,
  value: data.transactionValue,
};
```
Falsy values will already get excluded by the pixel.js.  There's no need for extra tests here.

Reply:
Oh, and there's a bug in the second test.  A zero value might be meaningful but is falsy.  We don't want to exclude it.

Reply:
Ahh true about the falsy value. OK I will change this back.